pkg/asset/manifests/infrastructure.go

```diff
@@ -311,6 +311,19 @@ func (i *Infrastructure) Generate(ctx context.Context, dependencies asset.Parent
 				URL:  service.URL,
 			})
 		}
+		if installConfig.Config.Publish == types.InternalPublishingStrategy &&
+			(len(installConfig.Config.ImageDigestSources) > 0 || len(installConfig.Config.DeprecatedImageContentSources) > 0) {
+			piRegion := installConfig.Config.PowerVS.Region
+			vpcRegion, err := powervs.VPCRegionForPowerVSRegion(piRegion)
+			if err != nil {
+				return errors.New("failed to determine VPC region")
```

Comment:
Are you not bubbling up the error on purpose?
```suggestion
				return fmt.Errorf("failed to determine VPC region: %w", err)
```

Reply:
Not on purpose. Overside. Fixed.